test-data/message/text_alt_plain_html.eml

```diff
@@ -0,0 +1,23 @@
+Subject: mime-modified test
```

Comment:
"HTML test"?

Reply:
i'd leave that as is, internally, i'll leave this mime-modified-flag.